db_utils.py

```diff
@@ -67,15 +67,24 @@ def connect(self):
 
```

Comment:
let me. know if there is a better way to do the self.attempts method , I am aware this may be hamfisted